dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataExport.java

```diff
@@ -88,4 +89,8 @@ public void setup() throws ParseException {
         }
         handle = commandLine.getOptionValue('i');
     }
+
+    protected String getFileNameForExportFile() {
+        return UUID.randomUUID().toString() + ".csv";
```

Comment:
No, please.  We should have a meaningful name for the export file & not a random filename.  I had recommended we name the export file based on the UUID of the collection/community you are exporting.  So, this name should be based on the `-i` input value (which specifies the object we are exporting).  It can be any one of the following:

* `[objection-uuid].csv`
* `[object-type]-[object-uuid].csv` (e.g. COLLECTION-[uuid].csv)
* Or, one of the above based on the value of `-i` param.  So, for example, if exporting a Collection with handle "123/456" then the file might be named: `123-456.csv` or `COLLECTION-123-456.csv`